build_requirements.txt

```diff
@@ -4,16 +4,16 @@
 #
 #    make update-deps
 #
-astroid==2.2.5            # via pylint
-click==7.0                # via pip-tools
-isort==4.3.18             # via pylint
-lazy-object-proxy==1.3.1  # via astroid
+astroid==2.4.2            # via pylint
+click==7.1.2              # via pip-tools
+isort==4.3.21             # via pylint
+lazy-object-proxy==1.4.3  # via astroid
 mccabe==0.6.1             # via pylint
-pathspec==0.5.9           # via yamllint
-pip-tools==3.6.1
-pylint==2.3.1
-pyyaml==5.1               # via yamllint
-six==1.12.0               # via astroid, pip-tools
-typed-ast==1.3.5          # via astroid
-wrapt==1.11.1             # via astroid
-yamllint==1.15.0
+pathspec==0.8.0           # via yamllint
+pip-tools==3.9.0
+pylint==2.5.3
```

Comment:
New pylint version gives new lint errors (`make release-test`) which are good to fix.

Reply:
After fixing the first few pylint errors (basically using sys.exit() as in https://github.com/SVT/orm/pull/47/commits/d69d09ce2366176b71a53b86dad2fb384791d5a5) pylint complained about all the module imports from the test scripts

```
lint tests
************* Module test_rendervarnish
test/test_rendervarnish.py:5:0: E0401: Unable to import 'orm.render' (import-error)
test/test_rendervarnish.py:6:0: E0401: Unable to import 'orm.rendervarnish' (import-error)
test/test_rendervarnish.py:7:0: E0401: Unable to import 'orm.rendervarnish' (import-error)
************* Module test_validator
test/test_validator.py:5:0: E0401: Unable to import 'orm.validator' (import-error)
************* Module test_parser
test/test_parser.py:6:0: E0401: Unable to import 'orm.parser' (import-error)
************* Module test_renderhaproxy
test/test_renderhaproxy.py:3:0: E0401: Unable to import 'orm.render' (import-error)
test/test_renderhaproxy.py:4:0: E0401: Unable to import 'orm.renderhaproxy' (import-error)
test/test_renderhaproxy.py:5:0: E0401: Unable to import 'orm.renderhaproxy' (import-error)
```
As described in https://github.com/SVT/orm/pull/47/commits/28c4d5c256b40f98772f0217bc160ee1bfc7f62c this is due to changes with how pylint imports modules. I dont see any trivial way to fix this which wouldnt be potentially overstepping the scope of this pull request, so maybe downgrade pylint?

Reply:
Sounds good